planning/planning_test_utils/README.md

```diff
@@ -1,92 +1,38 @@
-# Autoware Planning Test Manager
+# Planning Test Utils
 
 ## Background
 
-In each node of the planning module, when exceptional input, such as unusual routes or significantly deviated ego-position, is given, the node may not be prepared for such input and could crash. As a result, debugging node crashes can be time-consuming. For example, if an empty trajectory is given as input and it was not anticipated during implementation, the node might crash due to the unaddressed exceptional input when changes are merged, during scenario testing or while the system is running on an actual vehicle.
+Several planning components and modules have already adopted unit testing, so a common library to ease the process of writing unit tests is necessary.
 
 ## Purpose
 
-The purpose is to provide a utility for implementing tests to ensure that node operates correctly when receiving exceptional input. By utilizing this utility and implementing tests for exceptional input, the purpose is to reduce bugs that are only discovered when actually running the system, by requiring measures for exceptional input before merging PRs.
+The objective of the `planning_test_utils` is to develop a unit testing library for the planning components. This library will include
 
-## Features
+- commonly used functions
+- input/mock data parser
+- maps for testing
+- common routes and mock data for testing.
 
-### Confirmation of normal operation
+## Available Maps
 
-For the test target node, confirm that the node operates correctly and publishes the required messages for subsequent nodes. To do this, test_node publish the necessary messages and confirm that the node's output is being published.
+The following maps are available [here](https://github.com/autowarefoundation/autoware.universe/tree/main/planning/planning_test_utils/test_map)
 
-### Robustness confirmation for special inputs
+### Common
 
-After confirming normal operation, ensure that the test target node does not crash when given exceptional input. To do this, provide exceptional input from the test_node and confirm that the node does not crash.
+The common map contains multiple types of usable inputs, including shoulder lanes, intersections, and some regulatory elements. The common map is named `lanelet2_map.osm` in the folder.
 
-(WIP)
+![common](./images/common.png)
 
-## Usage
```

Comment:
@zulfaqar-azmi-t4 
I think it would be more user-friendly to include usage examples. Why were you considering removing them?
If the examples are of low quality, it might be better not to include them at all. Is that the reason behind the removal?

Reply:
I've added the link to the planning test manager 
https://github.com/autowarefoundation/autoware.universe/pull/7077/commits/be2d2575107577a35d91653219150460ae973e09

to show the example

https://github.com/autowarefoundation/autoware.universe/pull/7077/commits/be2d2575107577a35d91653219150460ae973e09